{{ cookiecutter.repo_name }}/plugins/{{ cookiecutter.plugin_name }}/{%- if cookiecutter.ui_type != "none" %}UI{{ cookiecutter.plugin_name }}.cpp{%- endif %}

```diff
@@ -0,0 +1,99 @@
+/*
+ * {{ cookiecutter.project_name }} audio effect based on DISTRHO Plugin Framework (DPF)
+ * Copyright (C) {{ cookiecutter.year }} {{ cookiecutter.author_name }} <{{ cookiecutter.email }}>
+ *
+ * Permission to use, copy, modify, and/or distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY
+ * SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION
+ * OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
+ * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ */
+
+#include "UI{{ cookiecutter.plugin_name }}.hpp"
+
+START_NAMESPACE_DISTRHO
+
```

Comment:
I'd like a few more method documentation comments in this file. Along the lines of https://github.com/DISTRHO/DPF/blob/master/examples/Meters/ExampleUIMeters.cpp#L50 ff.

Reply:
OK, I've added some documentation.